test/CMakeLists.txt

```diff
@@ -100,6 +97,16 @@ endif()
 
 target_link_libraries(gtest_zlib zlibstatic GTest::GTest)
 
+find_package(Threads)
+if(Threads_FOUND)
+	target_sources(gtest_zlib PRIVATE test_deflate_concurrency.cc)
```

Comment:
I think there is a tab character here. Sorry 😕

Reply:
Ouch. Thanks for noticing this.